pkg/skaffold/deploy/util.go

```diff
@@ -167,3 +168,8 @@ func ApplyDefaultRepo(globalConfig string, defaultRepo *string, tag string) (str
 
 	return newTag, nil
 }
+
+// ApplyRepoPrefix applies prefix to the given image tag
+func ApplyRepoPrefix(prefix string, image string) string {
```

Comment:
We'd presumably need a similar mechanism to `default-repo` to load values from the global config.